### DISCUSSION TITLE
What is 503 error in Vercel + NextJS?

### DISCUSSION THREAD

**Discussion by melodyxpot** [vercel/community#3838] [Help] [answered]
### Summary

Hi everyone, I made the project by myself and it was working well on my local.
Also I successfully deployed it on vercel.
But it was not working on deployment.
![image](https://github.com/vercel/community/assets/53822027/2b2a5880-e1da-41a7-b338-79d8274febd0)
I used app router and axios.post part is not working I think
This is my code that is not working on deployment
![image](https://github.com/vercel/community/assets/53822027/5c3b45e0-7771-4e49-af1d-1d6cb38e0593)
What is the problem?
Please help me

### Example

https://xipics.vercel.app/generate

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/53822027/b26da3a1-b8ca-427e-b083-bd964aa18f3e)
Just hit the Generate button in https://xipics.vercel.app/generate page

**Comment by amyegan** [comment, marked as answer]
Hey @melodyxpot. A [504](https://vercel.com/docs/errors/application-error-codes#504:-function_invocation_timeout) means the function reached the [timeout threshold](https://vercel.com/docs/limits/overview#serverless-function-execution-timeout). You can find more info about possible solutions at https://vercel.com/docs/errors/application-error-codes#504:-function_invocation_timeout
